docs/cardinal/game/configuration.mdx

```diff
@@ -0,0 +1,41 @@
+| BASE_SHARD_SEQUENCER_ADDRESS | The address of your EVM Base Shard deployment. This will typically be running on port 9601.                                                                                                                          | YES                     |
+
+## EVM Base Shard `[evm]`
+| Name            | Description                                                                                                                                                                                           |
+|-----------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| DA_AUTH_TOKEN   | The authentication token required to communicate with the Celestia DA Layer. This is typically generated by the celestia light client, and can be found in the logs during startup                    |
+| DA_BASE_URL     | The base URL of the DA layer.                                                                                                                                                                         |
+| DA_NAMESPACE_ID | The namespace the EVM will post its blobs under. Learn more about DA namespaces here [Namespaces](https://celestia.org/glossary/namespaced-merkle-tree/) Can be generated with `openssl rand -hex 10` |                                                                                                                 |
+| CHAIN_ID        | The chain identifier. This is used to prevent replay attacks across different chains.                                                                                                                 |
+| KEY_MNEMONIC    | The mnemonic of the account to instantiate the chain with. This will allocate some funds to the account.                                                                                              |
+| FAUCET_ADDR     | Supplying a faucet address enables the given account a supply of funds to use for faucet purposes.                                                                                                    |
+| BLOCK_TIME      | The interval at which the EVM Base Shard should produce blocks. Examples: `1s`, `2s`, `3s`...                                                                                                         |
+
+## Nakama Relayer `[nakama]`
+| Name                | Description                                                                                                                                 |
+|---------------------|---------------------------------------------------------------------------------------------------------------------------------------------|
+| ENABLE_ALLOWLIST    | Can be either **true** or **false**. Enables the allowlist functionality of Nakama. This gives Nakama endpoints to generate and claim keys. |
```

Comment:
We should have a section in our docs describing how to use the allow list feature. I made a ticket: https://linear.app/arguslabs/issue/WORLD-1052/document-allow-list-feature